userguide/intro.rst

```diff
@@ -223,12 +223,15 @@ These screen options have changed:
 * The :guilabel:`Hold Pending Snapshots` checkbox has been added to
   :menuselection:`Tasks --> Replication Tasks --> ADD`.
 
-* The :guilabel:`Use --fast-list` checkbox has been added to
-  :menuselection:`Tasks --> Cloud Sync Tasks --> ADD`.
-
 * The :guilabel:`Name` field has been added to
   :menuselection:`Tasks --> Replication Tasks --> Add`.
 
+* The default view has been adjusted in
```

Comment:
This should probably be more specific about what changed, if we can state it concisely.  Adjusted in what way?